src/js/service/startGame.js

```diff
@@ -0,0 +1,90 @@
+import RacingCar from '../model/RacingCar.js';
+import printSpinner from '../ui/printSpinner.js';
+import getRandomNumber from '../model/getRandomNumber.js';
+import { game } from '../constants.js';
+import renderCarStatus from '../ui/renderCarStatus.js';
+import hideSpinner from '../ui/hideSpinner.js';
+import { sortObjectByValue } from '../utils/index.js';
+
+const getWinnersName = (moveForwardCountArray) => {
+  const MOVE_FORWARD_CAR_NAME_INDEX = 0;
+  const MOVE_FORWARD_COUNT_INDEX = 1;
+
+  const firstWinner = moveForwardCountArray[0][MOVE_FORWARD_CAR_NAME_INDEX];
+  const winners = [firstWinner];
+
+  for (let i = 0; i < moveForwardCountArray.length - 1; i++) {
+    const currentWinner = moveForwardCountArray[i];
+    const nextWinner = moveForwardCountArray[i + 1];
+
+    if (
+      currentWinner[MOVE_FORWARD_COUNT_INDEX] > nextWinner[MOVE_FORWARD_COUNT_INDEX]
+    )
+      break;
+
+    winners.push(nextWinner[MOVE_FORWARD_CAR_NAME_INDEX]);
+  }
+
+  return winners;
+};
```

Comment:
```suggestion
const getWinnersName = (moveForwardCountArray) => {
  const max = Math.max(...moveForwardCountArray.map(([, count])=> count));
  return moveForwardCountArray.filter(([, count]) => count === max).map(([name]) => name);
};
```

이렇게 표현할 수 있겠네요.

그리고 이렇게 표현해보니까 moveForwardCountArray의 내부를 배열로 표현하니까 불편하다고 느껴지지 않나요!?